### PR TITLE
Add missing pdbx_database_status for MA models

### PR DESCRIPTION
**Pull request by brindakv**
This commit addresses the following: 
 - Add missing `pdbx_database_status` for MA models
 - Add `_rcsb_entry_info.structure_determination_methodology` for all entries
 - Update `pdbx_struct_assembly` for computed models

### CODE REVIEW

File: rcsb/utils/dictionary/DictMethodEntryHelper.py [hunk 1]
```diff
@@ -762,27 +774,28 @@ def addEntryInfo(self, dataContainer, catName, **kwargs):
...
+                if dataContainer.exists("ma_data"):
+                    methodType = "computational"
             #
             if dataContainer.exists("exptl"):
                 xObj = dataContainer.getObj("exptl")
                 entryId = xObj.getValue("entry_id", 0)
                 methodL = xObj.getAttributeValueList("method")
                 methodCount, expMethod = self.__commonU.filterExperimentalMethod(methodL)
-                if not methodType:
-                    methodType = self.__commonU.filterStructureDeterminationMethodType(methodL)
```
Comment (piehld) [inline rcsb/utils/dictionary/DictMethodEntryHelper.py:772]: Was there a strong reason to not use this method instead? This method was intended to also determine if it's integrative...

File: rcsb/utils/dictionary/DictMethodEntryHelper.py [hunk 2]
```diff
@@ -762,27 +774,28 @@ def addEntryInfo(self, dataContainer, catName, **kwargs):
                 xObj = dataContainer.getObj("struct")
                 if xObj.hasAttribute("pdbx_structure_determination_methodology"):
                     methodType = xObj.getValue("pdbx_structure_determination_methodology", 0)
+            if not methodType:
+                if dataContainer.exists("exptl"):
```
Comment (piehld) [inline rcsb/utils/dictionary/DictMethodEntryHelper.py:778]: Are we positive all experimental models in the archive will have `exptl`? (Even older ones)?

**Review by piehld — APPROVED** [review]
Looks good, but just a couple questions about implementation

File: rcsb/utils/dictionary/DictMethodEntryHelper.py (hunk 1 above)
Comment (brindakv) [reply rcsb/utils/dictionary/DictMethodEntryHelper.py:772]: Thought it is better to use a top-level category to determine if the entry is `experimental` or `computational`. Also, what if `_ma_model_list.model_type` is a `.` in the ModelCIF file?

File: rcsb/utils/dictionary/DictMethodEntryHelper.py (hunk 2 above)
Comment (brindakv) [reply rcsb/utils/dictionary/DictMethodEntryHelper.py:778]: I think so.

**Pull request merged**